docs/p/howto/Collective.rst

```diff
@@ -1,5 +1,351 @@
 Collective
 ==========
 
+Overview
+--------
+
 A collective effects element can be added to any lattice to model
-impedance driven collective effects and perform multi-particle tracking
+impedance driven collective effects and perform multi-particle tracking.
+This element can be constructed with either a user defined table, or by
+using the built in functions to construct specific elements (for example
+a longitudinal resonator or a transverse resistive wall). The element will
+then call the **WakeFieldPass** PassMethod. 
+
+The wake field is applied by first uniformly slicing the full region occupied by the 
+particles in the ct co-ordinate. Each particle is attributed to a
+given slice, which is represented by a weight. The mean position in x,y,ct 
+is computed for each slice. The kick from one slice to the next (and the self kick for some cases)
+can then be computed by taking into account the differences in offsets of each slice.
+This total kick is computed and the appropriate modification to x', y' or dp is applied
+to each particle. 
+
+To take into account multi-turn wakes, the wake element has a **TurnHistory** buffer.
+Each turn, the mean x,y,ct and weight of each slice is recorded. At each turn, the 
+array of ct values is increased by one circumference (to take into account the decay 
+between turns). When the kick is computed, the full history of turns to ensure the
+total kick from all previous turns is considered.
+
+
+The package is organised as follows:
+
+**at.collective.wake_functions** contains the analytic wake functions that can be called
+by the other classes
+
+The longitudinal resonator wake function is given by [1]
```

Comment:
You can still cross-reference the end note like this:
```
The longitudinal resonator wake function is given by [1]_
```
(underscore after the right bracket), 
And write the end note like:
```
.. [1] `A. Chao, 'Physics of Collective Beam Instabilities in High Energy Accelerators', p. 73, Eqn. 2.84 <https://www.slac.stanford.edu/~achao/WileyBook/WileyChapter2.pdf>`_.
```
Two dots and a space before the left bracket.

Reply:
thanks